nvflare/ha/ha_admin_cmds.py

```diff
@@ -66,10 +66,11 @@ def promote_sp(self, args, api):
             return {"status": APIStatus.ERROR_SYNTAX, "details": "usage: promote_sp example1.com:8002:8003"}
 
         sp_end_point = args[1]
-
-        print("PROMOTING SP: {}".format(sp_end_point))
         api.overseer_agent.promote_sp(sp_end_point)
-        return {"status": APIStatus.SUCCESS, "details": "Promoted endpoint. Synchronizing with overseer..."}
+        return {
+            "status": APIStatus.SUCCESS,
+            "details": "Attempted to promote endpoint: {}. Synchronizing with overseer...".format(sp_end_point),
+        }
```

Comment:
We should be able to know if the promote is successful or not?
Should we reflect that here in the details?